Assets/Scripts/Combat/Abilities/Confection.cs

```diff
@@ -46,7 +46,7 @@ public new void Start()
         base.Start();
         Timer = GetComponent<Timer>();
         
-        StartUI();
+        //StartUI();
```

Comment:
Why is that commented?

Reply:
We didn't set up the cooking ability in the main scene. So we commented out that bit cause we were getting errors.

Reply:
We should fix this now actually so it works

Reply:
Doing it right now.